docs/tutorials/creating-your-first-substrate-chain/interact.md

```diff
@@ -76,13 +73,12 @@ Amount: 1000
 
 ![Balance Transfer](assets/front-end-template-balance-transfer.png)
 
-If everything went successfully, you should see some notifications in the
-**Events** component, and of course Charlie's balance will now be `1000`.
+If everything went successfully, you should see some notifications in the **Events** component, and
+of course Charlie's balance will now be `1000`.
 
-Already you have a working blockchain, with an underlying cryptocurrency. You
-are able to make transfers easily with a simple, interactive front-end.
+Already you have a working blockchain, with an underlying cryptocurrency. You are able to make
+transfers easily with a simple, interactive front-end.
 
-Now let's build our Proof of Existence Pallet
+Now let's build our Proof of Existence pallet!
```

Comment:
Why are we using `pallet` and not `Pallet`. Isn't Pallet a proper noun? Similar to `Containers`, `Crates`?
Not an important change but important to think about it.